css/style.css

```diff
@@ -30,7 +41,172 @@
     float: right;
     margin-top: 50px;
     border: 1px solid grey;
-    padding: 10px;
+    padding: 10px 15px;
+    font-size: 17px;
     background: #2196F3;
     color: white;
+}
+
+
+/*Slider one*/
+
+.site-slider {
+    position: relative;
+}
+
+.position-top {
+    position: absolute;
+    top: 50%;
+}
+
+.site-slider .slider-btn .prev,
+.site-slider .slider-btn .next {
+    background-color: #6983aa;
+    padding: 10px 20px;
+    border-radius: 100%;
+    color: white;
+    margin: 0 10px;
+    opacity: 0;
+    transition: opacity 1s ease;
+}
+
+.site-slider:hover .slider-btn .prev,
+.site-slider:hover .slider-btn .next {
+    opacity: 1;
+}
+
+.right-0 {
+    right: 0;
+}
+
+
+/*SLick Dots*/
+
+.slick-dots {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    padding: 10px 0;
+    list-style-type: none;
+    position: absolute;
+    bottom: 0;
+    right: 50%;
```

Comment:
Move this on top of slick-dots. It should be talking from outside to inside

Reply:
done